components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth/dao/OAuthAppDAO.java

```diff
@@ -944,11 +980,61 @@ private boolean isUsernameCaseSensitive(String tenantQualifiedUsername) {
      *
      * @param tenantDomain application tenant domain
      * @param consumerKey  client ID
-     * @return
+     * @return idTokenAudiences audience values for Id Token
      * @throws IdentityOAuth2Exception
      */
+    /**
+     * @deprecated use {@link #getOIDCIdTokenAudiences(String, String )} |
+     * {@link #getOIDCAccessokenAudiences(String, String )} instead.
```

Comment:
typo in `getOIDCAccessokenAudiences`

Reply:
fixed with 95271dc3c7cc820ee8b77a92ce75c0129db0015b